docs/DeveloperGuide.md

```diff
@@ -444,18 +442,18 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
     Use case ends.
 
-<ins>**Use case: UC09 - Add member to an event**</ins>
+<ins>**Use case: UC09 - Enrol member to an event**</ins>
 
-**Guarantees: MSS -> Specified member will be added to the event.**
+**Guarantees: MSS -> Specified member will be enroled to the event.**
```

Comment:
```suggestion
**Guarantees: MSS -> Specified member will be enrolled to the event.**
```
Minor typo here.

Reply:
Oh right, enrolled is the correct spelling